deployment/Service-Name-Installation/docker-compose.yml

```diff
@@ -146,78 +175,89 @@ services:
       - WEBSOCKET_ENCRYPT_KEY=${WEBSOCKET_ENCRYPT_KEY:-giert989jkwrgb@DR55}
       - DATA_BUFFER_SIZE=${DATA_BUFFER_SIZE:-2}
       - IDENTITY_PROVIDER_MAX_RESULT_SIZE=${IDENTITY_PROVIDER_MAX_RESULT_SIZE:-250}
+      - KEYCLOAK_WEB_CLIENTID=${KEYCLOAK_WEB_CLIENTID:-}
+      - KEYCLOAK_ENABLE_CLIENT_AUTH=${KEYCLOAK_ENABLE_CLIENT_AUTH:-false}
+      - BPM_CLIENT_CONN_TIMEOUT=${BPM_CLIENT_CONN_TIMEOUT:-5000}
       - DATA_ANALYSIS_URL=${DATA_ANALYSIS_URL}
       - CUSTOM_SUBMISSION_URL=${CUSTOM_SUBMISSION_URL}
       - CUSTOM_SUBMISSION_ENABLED=${CUSTOM_SUBMISSION_ENABLED:-false}
+      - MULTI_TENANCY_ENABLED=${MULTI_TENANCY_ENABLED:-false}
+      - FORMSFLOW_ADMIN_URL=${FORMSFLOW_ADMIN_URL:-}
+      - REDIS_ENABLED=${REDIS_ENABLED:-false}
+      - REDIS_HOST=${REDIS_HOST}
+      - REDIS_PORT=${REDIS_PORT:-6379}
+      - REDIS_PASSCODE=${REDIS_PASSCODE:-changeme}
+      - SESSION_COOKIE_SECURE=${SESSION_COOKIE_SECURE:-false} 
     networks:
       - forms-flow-network
   forms-flow-webapi:
     build:
       context: ../../forms-flow-api/
       dockerfile: Dockerfile
-    restart: always
     container_name: forms-flow-webapi
-    ports:
-      - 5000:5000
-    volumes:
-      - ../../forms-flow-api:/app:rw
+    restart: unless-stopped
     depends_on:
-      - forms-flow-bpm-db
-      - forms-flow-bpm
+      - keycloak
       - forms-flow-forms
+      - forms-flow-bpm
+      - keycloak_db
+      - forms-flow-forms-db
+      - forms-flow-bpm-db
       - forms-flow-webapi-db
 
+    entrypoint: "/wait-for-service.sh forms-flow-webapi-db:5432 -s -- ./entrypoint.sh"
```

Comment:
Here we can do recursive wait like here : https://github.com/AOT-Technologies/forms-flow-ai-deployment/blob/main/docker-compose/docker-compose.yml#L141

Also shouldn't this be in forms-flow-ai-deployment repo as all deployments in this repo are only for local development ?

Reply:
Yes, better that way.